### PR TITLE
Fix: push docker images to Alibaba Cloud ACR

### PR DESCRIPTION
**Pull request by zzxwill**
Added Github action to push docker images to Alibaba Cloud ACR

### Description of your changes

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

File: .github/workflows/registry.yml [hunk 1]
```diff
@@ -44,6 +44,9 @@ jobs:
           registry: docker.io
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login Alibaba Cloud ACR
+        run: |
```
Comment (zzxwill) [inline .github/workflows/registry.yml:48]: Replace it with https://github.com/marketplace/actions/alibaba-cloud-container-registry-acr-login

File: .github/workflows/registry.yml [hunk 2]
```diff
@@ -80,6 +84,7 @@ jobs:
           tags: |-
             ghcr.io/${{ github.repository }}/vela-rollout:${{ steps.get_version.outputs.VERSION }}
             docker.io/oamdev/vela-rollout:${{ steps.get_version.outputs.VERSION }}
+            kubevela-registry.cn-hangzhou.cr.aliyuncs.com/oamdev/vela-rollout:${{ steps.get_version.outputs.VERSION }}
```
Comment (wonderflow) [inline .github/workflows/registry.yml:87]: push 的顺序改下，先push docker hub，再部署 acr，最后 ghcr

File: .github/workflows/registry.yml [hunk 3]
```diff
@@ -44,6 +44,9 @@ jobs:
           registry: docker.io
           username: ${{ secrets.DOCKER_USERNAME }}
           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Login Alibaba Cloud ACR
+        run: |
+          docker login -u ${{ secrets.ACR_USERNAME }}@aliyun-inner.com acr.kubevela.net -p ${{ secrets.ACR_PASSWORD }}
```
Comment (wonderflow) [inline .github/workflows/registry.yml:49]:
```suggestion
          docker login -u ${{ secrets.ACR_USERNAME }}@aliyun-inner.com kubevela-registry.cn-hangzhou.cr.aliyuncs.com -p ${{ secrets.ACR_PASSWORD }}
```

域名改下，推送用原来的。

**Comment by zzxwill** [comment]
Verified.

image

**Review by hongchaodeng — APPROVED** [review]

**Pull request merged**